utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/util/json/JsonSchemaImplicits.scala

```diff
@@ -12,6 +12,11 @@ object JsonSchemaImplicits {
 
   implicit class ExtendedSchema(schema: Schema) {
 
+    def representsMap: Boolean = schema match {
```

Comment:
isMapSchema? :)

Reply:
and what's wrong with `representsMap`? The point of this method is to check whether a schema should be translated to Map type in NU. Maybe in the future this implementation will evolve and we will check another conditions, not only `addionoalProperties` - that's why I don't like `hasAdditionalPropertiesOnly`

Reply:
ok. But having a name `representsMap` has allowed me to not write comments why I handle differently this schema when typing. Now it might be not as understandable to a reader. But ok. let's leave it and move on.

Reply:
For me it is the metter where this method is, it is part of "our" logic or "json schema logic". Do we added some extensions for easier handling json schemas that are related with json schema or it is our specific extensions. If it is our specific extensions we should name class otherwise, place it closer to place of usage etc. If its are "neutral" schema extensions, we shouldn't introduce some specific logic. TL;DR for me it could be fine to call it `representsMap` but rename class / move class then.